js/recruitmentInfo.js

```diff
@@ -0,0 +1,3 @@
+export let recruitmentInfoRequest =
+  fetch('../api/rekruteringsinfo/')
```

Comment:
Kan vi anta at API-et alltid vil leve på root av domenet som `/api/*`? I så fall kan du droppe `..` og så vil dette fungere selv denne filen brukes i ulike subfolders? Jeg var kanskje litt utydelig tidligere, men med relativt mente jeg at du ikke inkluderte origin delen av url-en. Mens det git fortsatt mening å bruke absolute path slik at man slipper å forholde seg til hvilken mappe ting ligger i.

Reply:
Ah, så en addresse typ "/api/rekruteringsinfo/" tolkes som <origin>/api/rekruteringsinfo? Var usikker på åssen dette funker for nettadresser, men ja, synes det er en ok antagelse at /api/ lever på rotnivå